source/sass/patterns/molecules/contextual-data.scss

```diff
@@ -0,0 +1,27 @@
+@import "../../mixins/decorations";
+@import "../../mixins/positioning";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+@import "../../settings/baselinegrid";
+@import "../../settings/color";
+
+.contextual-data {
+  @include block-spacing($end: $baselinegrid-space-medium);
+  @include border(block-end);
+  @include padding($baselinegrid-space-extra_small, block);
+  @include text-align(center);
+  font-family: $font-secondary;
```

Comment:
This should use `label-content-typography` instead. Means that no text-styling needs to be applied to the DOI link (_apart_ from `text-transform: none`).

Refs https://github.com/libero/pattern-library/pull/131#discussion_r310546969.